html/pfappserver/lib/pfappserver/Form/NodeSearch.pm

```diff
@@ -0,0 +1,75 @@
+package pfappserver::Form::NodeSearch;
+
+=head1 NAME
+
+pfappserver::Form::Node - Web form for a node
+pfappserver::Form::Search::Node - Web form for searching Nodes
+
+=head1 DESCRIPTION
+
+Form definition to create or update a node.
```

Comment:
Didn't know you could update the nodes directly from the search bar.

PacketFence is getting so powerful